fixtures/chauvet-dj/corepar-uv-usb.json

```diff
@@ -0,0 +1,170 @@
+{
+  "$schema": "https://raw.githubusercontent.com/FloEdelmann/open-fixture-library/master/schemas/fixture.json",
+  "name": "COREpar UV USB",
+  "shortName": "ChauvetCOREparUV",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["Felix Edelmann"],
+    "createDate": "2018-03-15",
+    "lastModifyDate": "2018-03-15"
+  },
+  "manualURL": "https://www.chauvetdj.com/wp-content/uploads/2015/12/COREpar_UV_USB_UM_Rev1_WO-1.pdf",
+  "physical": {
+    "dimensions": [200, 253, 139],
+    "weight": 2.1,
+    "power": 65,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "70W COB LED"
+    },
+    "lens": {
+      "name": "Fresnel",
+      "degreesMinMax": [50, 100]
+    },
+    "focus": {
+      "type": "Fixed"
+    }
+  },
+  "availableChannels": {
+    "UV": {
+      "type": "Single Color",
+      "color": "UV",
+      "defaultValue": 0
+    },
+    "Strobe": {
+      "type": "Strobe",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "range": [0, 15],
+          "name": "No function"
+        },
+        {
+          "range": [16, 255],
+          "name": "Slow-fast"
+        }
+      ]
+    },
+    "Program Speed": {
+      "type": "Speed",
+      "capabilities": [
+        {
+          "range": [0, 255],
+          "name": "Slow-fast"
+        }
+      ]
+    },
+    "Sound Sensitivity": {
+      "type": "Intensity",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "range": [0, 10],
+          "name": "Off"
+        },
+        {
+          "range": [11, 255],
+          "name": "Sound sensitivity low-high"
+        }
+      ]
+    },
+    "Programs": {
+      "type": "Effect",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "range": [0, 31],
+          "name": "No function",
+          "switchChannels": {
+            "Strobe / Program Speed / Sound Sensitivity": "Strobe"
+          }
+        },
+        {
+          "range": [32, 95],
+          "name": "Pulse effect",
```

Comment:
In the manual, the three pulse effect capabilities have some more description with percentages. Please add them here.